code/index.html

```diff
@@ -4,19 +4,169 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+
+           <div class="navigationbar"> <nav>
+          <a href="#home" class="active"><picture>
+            <img src="./logo.svg" alt="technigo">
+           </picture></a>
+          <div id="myLinks">
+           <a href="#">SIGTHSEEING</a>
+           <a href="#">RESTAURANTS</a>        
+           <a href="#">BARS AND CLUBS</a>
+           <a href="#">CULTURE</a>
+           <a href="#">EVENTS</a>
+          </div>
+          <a href="javascript:void(0);" class="icon" onclick="myFunction( )">
+            <i class="fa fa-bars"></i>
+          </a>
+          </nav>
+          </div>
+
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+    <section class="large-news">
+      <div><picture>
+       <img src="Berlin.jpg" alt="berlin">
+      </picture>
+    </div>
+    <div>
+      <h2>
+        Discover Berlin!
+      </h2>
+      <p>
+        Berlin is since decades a own cosmoplitan. The city changes so quickly, it´s hard to keep update. We want you to keep it in the loop about what is happening in the world of music, culture and food. Keep posted!
+      </p>
+    </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="news">
+
+     <div class="small-news">
+      <picture>
+        <img src="coffeeshop.jpg" alt="coffeeshop">
```

Comment:
The alt attribute could be more descriptive. This increases accessibility of your website. For example "two cups of coffee" would give the viewer a better description.